zerver/views/typing.py

```diff
@@ -0,0 +1,20 @@
+from django.http import HttpRequest, HttpResponse
+from six import text_type
+
+from zerver.decorator import authenticated_json_post_view,\
+    has_request_variables, REQ, JsonableError
+from zerver.lib.actions import check_send_typing_notification, \
+    extract_recipients
+from zerver.lib.response import json_success
+from zerver.models import UserProfile
+
+@has_request_variables
+def send_typing_notification_backend(request, user_profile,
+                                     operator = REQ('op'),
+                                     notification_to = REQ('to', \
+                                            converter=extract_recipients, \
+                                            default=[])):
+    # type: (HttpRequest, UserProfile, text_type, List[text_type]) -> HttpResponse
+    sender = user_profile
```

Comment:
This variable is unnecessary, I'd just pass `user_profile` into the check function below.